test/test_playbooks/product.yml

```diff
@@ -1,22 +1,46 @@
 ---
-- hosts: localhost
+- hosts: fixtures
+  gather_facts: false
   tasks:
   - name: 'Load server config'
     include_vars:
       file: server_vars.yml
   - include: tasks/organization.yml
     vars:
       organization_state: present
+- hosts: tests
+  gather_facts: false
+  tasks:
+  - name: 'Load server config'
+    include_vars:
+      file: server_vars.yml
+  - include: tasks/product.yml
+    vars:
+      product_state: present
+      expected_change: true
```

Comment:
Very nice.

Reply:
What does expected_change do?

Reply:
it's used in the playbook to verify that the state of the object was or was not changed: https://github.com/theforeman/foreman-ansible-modules/pull/76/files#diff-f5f51acb20b549427b6bc248abf7cc95R14